.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
Yeah, though given that this is only a library plugin, it seems unlikely someone would `hpi:run` directly from here. Possible. Can always be added later.